diff -ur bridge-utils-1.5.orig/libbridge/libbridge.h bridge-utils-1.5/libbridge/libbridge.h
--- bridge-utils-1.5.orig/libbridge/libbridge.h	2011-03-29 00:52:54.000000000 +0000
+++ bridge-utils-1.5/libbridge/libbridge.h	2014-07-18 16:25:44.836000000 +0000
@@ -21,6 +21,7 @@
 
 #include <sys/socket.h>
 #include <linux/if.h>
+#include <linux/in6.h>
 #include <linux/if_bridge.h>
 
 /* defined in net/if.h but that conflicts with linux/if.h... */
