--- bridge-utils-1.5.orig/brctl/Makefile.in
+++ bridge-utils-1.5/brctl/Makefile.in
@@ -37,7 +37,7 @@
 	$(CC) $(LDFLAGS) $(brctl_OBJECTS) $(LIBS) -o brctl
 
 %.o: %.c brctl.h
-	$(CC) $(CFLAGS) $(INCLUDE) -c $< 
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(INCLUDE) -c $< 
 
 clean:
 	rm -f *.o brctl core
--- bridge-utils-1.7.1/libbridge/Makefile.in.orig	2021-03-04 18:27:42.000000000 +0100
+++ bridge-utils-1.7.1/libbridge/Makefile.in	2021-05-08 22:34:34.021409124 +0200
@@ -34,8 +34,8 @@
 	$(RANLIB) $@
 
 %.o: %.c libbridge.h libbridge_private.h
-	$(CC) $(CFLAGS) $(INCLUDE) -c $<
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(INCLUDE) -c $<
 
 libbridge_compat.o:	libbridge_compat.c if_index.c
-	$(CC) $(CFLAGS) -c libbridge_compat.c
+	$(CC) $(CFLAGS) $(CPPFLAGS) -c libbridge_compat.c
 
