--- bridge-utils-1.5.orig/brctl/Makefile.in
+++ bridge-utils-1.5/brctl/Makefile.in
@@ -37,7 +37,7 @@
 	$(CC) $(LDFLAGS) $(brctl_OBJECTS) $(LIBS) -o brctl
 
 %.o: %.c brctl.h
-	$(CC) $(CFLAGS) $(INCLUDE) -c $< 
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(INCLUDE) -c $< 
 
 clean:
 	rm -f *.o brctl core
--- bridge-utils-1.5.orig/libbridge/Makefile.in
+++ bridge-utils-1.5/libbridge/Makefile.in
@@ -5,8 +5,7 @@
 RANLIB=@RANLIB@
 
 CC=@CC@
-CFLAGS = -Wall -g $(KERNEL_HEADERS)
-
+CFLAGS = -Wall -g $(KERNEL_HEADERS) @CFLAGS@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 includedir=@includedir@
@@ -34,8 +33,8 @@
 	$(RANLIB) $@
 
 %.o: %.c libbridge.h libbridge_private.h
-	$(CC) $(CFLAGS) $(INCLUDE) -c $<
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(INCLUDE) -c $<
 
 libbridge_compat.o:	libbridge_compat.c if_index.c
-	$(CC) $(CFLAGS) -c libbridge_compat.c
+	$(CC) $(CFLAGS) $(CPPFLAGS) -c libbridge_compat.c
 
