diff -ur bridge-utils-1.5.orig/Makefile.in bridge-utils-1.5/Makefile.in
--- bridge-utils-1.5.orig/Makefile.in	2011-03-29 00:52:54.000000000 +0000
+++ bridge-utils-1.5/Makefile.in	2014-07-18 16:22:35.920000000 +0000
@@ -14,10 +14,10 @@
 SUBDIRS=libbridge brctl doc
 
 all:
-	for x in $(SUBDIRS); do $(MAKE) $(MFLAGS) -C $$x ; done
+	for x in $(SUBDIRS); do $(MAKE) $(MFLAGS) -C $$x || exit 1 ; done
 
 clean:
-	for x in $(SUBDIRS); do $(MAKE) $(MFLAGS) -C $$x clean ; done
+	for x in $(SUBDIRS); do $(MAKE) $(MFLAGS) -C $$x clean || exit 1 ; done
 
 distclean:	clean
 	rm -f config.log
@@ -31,5 +31,5 @@
 	rm -f doc/Makefile
 
 install:
-	for x in $(SUBDIRS); do $(MAKE) $(MFLAGS) -C $$x install; done
+	for x in $(SUBDIRS); do $(MAKE) $(MFLAGS) -C $$x install || exit 1 ; done
 
